app/src/main/java/com/sopt/shinmungo/presentation/gallery/GalleryDetailScreen.kt

```diff
@@ -0,0 +1,169 @@
+import androidx.compose.ui.graphics.vector.ImageVector
+import androidx.compose.ui.layout.ContentScale
+import androidx.compose.ui.res.painterResource
+import androidx.compose.ui.res.stringResource
+import androidx.compose.ui.res.vectorResource
+import androidx.compose.ui.tooling.preview.Preview
+import androidx.compose.ui.unit.dp
+import androidx.lifecycle.viewmodel.compose.viewModel
+import androidx.lifecycle.compose.collectAsStateWithLifecycle
+import coil.compose.AsyncImage
+import com.sopt.shinmungo.R
+import com.sopt.shinmungo.core.designsystem.component.topbar.CommonTopBar
+import com.sopt.shinmungo.core.designsystem.theme.ShinMunGoTheme
+import com.sopt.shinmungo.domain.entity.Photo
+import java.time.LocalDate
+import java.time.LocalDateTime
+import java.time.format.DateTimeFormatter
+
+@Composable
+fun GalleryDetailScreen(
+    modifier: Modifier = Modifier,
+    photoId: Int,
+    onBackClick: () -> Unit,
+    viewModel: GalleryViewModel = viewModel()
+) {
+    val uiState by viewModel.state.collectAsStateWithLifecycle()
+    val photo = uiState.photos.values.flatten().find { it.id == photoId }
+
+    Column(
+        modifier = modifier
+            .fillMaxSize()
+            .background(Color.White)
+    ) {
+        CommonTopBar(
+            title = stringResource(R.string.gallery_detail_topbar_title),
+            onLeftContent = {
+                IconButton(onClick = onBackClick) {
+                    Icon(
+                        imageVector = ImageVector.vectorResource(R.drawable.ic_arrow_left_line_white_24),
+                        contentDescription = stringResource(R.string.back_screen_text),
+                        tint = Color.White
+                    )
+                }
+            }
+        )
+
+        Box(
+            modifier = Modifier
+                .weight(1f)
+                .fillMaxWidth(),
+            contentAlignment = Alignment.Center
+        ) {
+            Column(modifier = Modifier.fillMaxWidth()){
+                AsyncImage(
+                    model = photo?.url,
+                    contentDescription = null,
+                    modifier = Modifier.fillMaxWidth(),
+                    contentScale = ContentScale.FillWidth
+                )
+                Row(
+                    modifier = Modifier
+                        .fillMaxWidth()
+                        .padding(8.dp),
+                    horizontalArrangement = Arrangement.SpaceBetween
+                ) {
+                    Box(
+                        modifier = Modifier
+                            .padding(4.dp)
+                            .border(1.dp, ShinMunGoTheme.color.gray3, RoundedCornerShape(4.dp))
+                    ){
+                        Text(
+                            modifier = Modifier.padding(4.dp),
+                            text = stringResource(R.string.gallery_detail_timestamp_prefix_text) + photo?.timestamp?.format(DateTimeFormatter.ofPattern("yyyy/MM/dd    HH:mm:ss")) ?: "",
```

Comment:
문자열 리소스에 `%s`와 같은 형식자를 넣어서 stringResource를 사용할 때 실행인자로 넘겨줄 수 있습니다!
```suggestion
                            text = stringResource(R.string.gallery_detail_timestamp_prefix_text, photo?.timestamp?.format(DateTimeFormatter.ofPattern("yyyy/MM/dd    HH:mm:ss")) ?: ""),
```
그리고 여기에서도 형식자를 상수로 관리해주면 코드 가독성과 유지보수성이 개선될 것이라고 생각합니다!!